src/equations/ideal_mhd_multiion_2d.jl

```diff
@@ -223,6 +223,91 @@ function source_terms_standard(u, x, t, equations::IdealMhdMultiIonEquations2D)
   return SVector(s)
 end
 
+    """
+    Collision source term, cf. Rueda-Ramirez et al. (2023) and Rubin et al. (2015)
+    """
+    function source_terms_collision(u, x, t, equations::IdealMhdMultiIonEquations2D)        
+        S_std = source_terms_standard(u, x ,t, equations)
+
+        s = zero(MVector{nvariables(equations), eltype(u)})
+        @unpack gammas = equations
+        @unpack charge_to_mass = equations
+        
+        v1_plus, v2_plus, v3_plus, _ = charge_averaged_velocities(u, equations)
+
+        total_electron_charge = zero(u[1])
+        for k in eachcomponent(equations)
+            rho_k = u[(k-1)*5+4]
+            total_electron_charge +=  rho_k * charge_to_mass[k]
+        end
+        
+        p = pressure(u, equations)
+        m_O_plus = 10 #TODO! 
+        m_O2_plus = 20 # TODO!
+        m = [m_O_plus, m_O2_plus] # todo mass vector to equations
+        
+        e = 1.602176634e−19
+        m_e = 9.109e−31 
+        n_e = total_electron_charge
+        alpha = 0.75 # TODO save in equations
+        p_e = alpha * sum(p) # TODO generalize and use in non-conservative flux!
+        T_e = p_e / (n_e * m_e) 
+        k_B = 1.380649e−23
```

Comment:
We have to find a smart normalization to avoid these physical constants in SI units